docs/tech/system/admin-panel.md

```diff
@@ -4,13 +4,84 @@ sidebar_position: 4
+
+## Repo 📦
+All the code related to Admin Panel can be accessed through
+https://github.com/Samarth-HP/admin-ts
+
+## Features ✨
+
+### View details associated with any of the 7 tabs available.
+
+![view-details](https://github.com/Samarth-HP/admin-ts/assets/46066481/b2d4c301-ea84-4723-bc1c-93a60ae265a1)
+
+### Ability to create new entries easily
+![create-new-entries](https://github.com/Samarth-HP/admin-ts/assets/46066481/9e1326c2-a77d-408a-bf17-1d3b962f4d7f)
+
+### Edit any information as and when required
+
+![edit-details](https://github.com/Samarth-HP/admin-ts/assets/46066481/d4c9d82b-3952-49ea-9e3e-6de19253c077)
+
+### Filters to reduce the noise :)
+![filter-data](https://github.com/Samarth-HP/admin-ts/assets/46066481/0390bd93-df2a-4640-a73b-1559c558e81e)
+
+### One click export to PDF
+![export-to-pdf](https://github.com/Samarth-HP/admin-ts/assets/46066481/a4ee7e87-441e-4bf4-bda6-90f3726504aa)
+
+### Delete unwanted entries
+![delete-data](https://github.com/Samarth-HP/admin-ts/assets/46066481/53fad1ee-796f-4e5f-8d5d-540b5d8cd74d)
+
+
+## Working ⚙️
+
+![adminpanel drawio (1)](https://github.com/Samarth-HP/admin-ts/assets/46066481/f4a1505a-ba41-450f-9f30-30fadd665268)
+
+1. Within the React App, users interact with the user interface to select or modify information. This can include actions such as updating a student's details, adding a new teacher, or modifying school information.
+
+2. To handle these user actions, a Hasura Data Provider, like ra-data-provider, is employed. This data provider acts as a bridge between the React App and the Hasura API. It facilitates the conversion of user actions into GraphQL queries, which define the desired data retrieval or modification operations.
+
+3. The GraphQL queries, generated by the Hasura Data Provider, are sent to the Hasura API. The API acts as the middleware, receiving the queries and processing them accordingly.
+
+4. Upon receiving the GraphQL queries, the Hasura API performs the necessary transformations. It converts the GraphQL queries into SQL queries, which are compatible with the underlying database. These SQL queries are then executed to retrieve or modify the data as requested by the user.
+
+5. Once the data is retrieved or modified, it is sent back from the Hasura API to the Data Provider. The Data Provider then processes the data and prepares it to be sent back to the User Interface.
+
+6. Finally, the data is delivered back to the User Interface within the React App. The user interface can then update accordingly, displaying the requested data modifications or presenting the retrieved information to the user.
+
+
+## Flows
+
+Currently Admin Panel supports 7 different tabs namely 
+- Student
```

Comment:
Can we link them to the code that controls this?

Reply:
Should I add the sidebar ka snippet or something? Or something from app.js?